CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ CHANGELOG
 
 ## HEAD (Unreleased)
 
+- Add information about in-flight operations to `pulumi stack`.
```

Comment:
You should back this change out. Let's update the `CHANGELOG.md` file in the PR that actually hooks up the feature. This PR just adds the necessary wiring, but doesn't actually add any information yet.